### ISSUE TITLE
Accuracy of Naive Bayes classifier

### ISSUE THREAD

**Issue by chandra589** [sloria/TextBlob#128]
Hi,
    Does anyone know how to get the accuracy, precision, recall and most informative features of Naive Bayes Classifier which is already trained using movie corpus data.

Thank you

**Comment by IvRRimum** [comment]
http://textblob.readthedocs.io/en/dev/classifiers.html#evaluating-classifiers

**Comment by ikmalpani** [comment]
This is only for the accuracy. What about precision, recall, and f-score?

Thanks.

**Comment by radityagumay** [comment]
Hi, any answer for this?

**Comment by mynameisvinn** [comment]
if you have a set of (a) predictions and (b) ground truth, you can calculate precision, recall, etc. by hand or by sklearn.

for example:
```python
from sklearn.metrics import precision_score
print precision_score(y_true, y_pred)  # automatically calculates precision
```

**Comment by sloria** [comment]
Closing this, as it has gone stale. Feel free to re-open if this is still relevant.
